src/component/utils/api.js

```diff
@@ -21,4 +21,12 @@ export const getCommentsByArticleId = (article_id) => {
   return newsApi.get(`/articles/${article_id}/comments`).then(({ data }) => {
     return data.articleComments;
   });
-};
\ No newline at end of file
+};
+
+export const patchArticleUpvote = (article_id) => {
+  return newsApi.patch(`/articles/${article_id}`, { inc_votes: 1 });
+};
+
+export const patchArticleDownvote = (article_id) => {
+  return newsApi.patch(`/articles/${article_id}`, { inc_votes: -1 });
+};
```

Comment:
Nice patch functions, though these could be refactored into one `patchArticleVote` function, which also takes in an  value to increment the votes by. (`particleArticleVote(article_id,incrementVote)`)

Reply:
Done  :white_check_mark: